### PR TITLE
make ff-subplot doc python3 compatible

### PR DESCRIPTION
**Pull request by Kully**
resolves: https://github.com/plotly/documentation/issues/1107

### CODE REVIEW

**Comment by Kully** [comment]
@cldougl

File: _posts/python/subplots/ff-subplots/2015-06-30-ff-subplots.html [hunk 1]
```diff
@@ -309,10 +331,14 @@ <h4 id="Horizontal-Table-and-Chart">Horizontal Table and Chart<a class="anchor-l
...
-<div class="output_area"><div class="prompt output_prompt">Out[8]:</div>
+<div class="output_area">
+
+<div class="prompt output_prompt">Out[7]:</div>
+
+
 
 <div class="output_html rendered_html output_subarea output_execute_result">
-<iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://plot.ly/~PythonPlotBot/461.embed" height="530px" width="100%"></iframe>
+<iframe id="igraph" scrolling="no" style="border:none;" seamless="seamless" src="https://plot.ly/~jordanpeterson/1337.embed" height="530px" width="100%"></iframe>
```
Comment (cldougl) [inline _posts/python/subplots/ff-subplots/2015-06-30-ff-subplots.html]: @Kully in the future let's all use the`PythonPlotBot` user for these docs

File: _posts/python/subplots/ff-subplots/ff-subplots.ipynb [hunk 2]
```diff
@@ -158,7 +163,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Add the data and layout objects:"
+    "Combine the data and layout objects to a new figure"
```
Comment (cldougl) [inline _posts/python/subplots/ff-subplots/ff-subplots.ipynb]: I would say:
- Combine the data and layout objects: 
or 
- Combine the data and layout objects to create a figure

File: _posts/python/subplots/ff-subplots/2015-06-30-ff-subplots.html (hunk 1 above)
Comment (Kully) [reply _posts/python/subplots/ff-subplots/2015-06-30-ff-subplots.html]: im rerunning the examples with the PlotBot account

**Pull request merged**